101-create-win-vm-2nic-existingvnet/azuredeploy.parameters.json

```diff
@@ -0,0 +1,73 @@
+"parameters": {
+    "vmname": {
+      "type": "string",
+      "minLength": 1,
```

Comment:
parameter files don't have types/minLengths/metadata; they have values; please supply sample values in accordance with the contribution guide section on parameter placedholders: https://github.com/Azure/azure-quickstart-templates/tree/master/1-CONTRIBUTION-GUIDE